README.md

```diff
@@ -65,8 +65,56 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
 
 **CA Cluster Issuer Sample values.yaml**
 - Download latest sample values.yaml: [values.yaml for
-  v0.0.1](caclusterissuer/values.yml)
+  v0.0.1](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=ca-clusterissuer/v0.0.1/ca-clusterissuer-data-values.yml)
+
+
+## Cloud Native Registry
+
+</br>
+</br>
+<p align="left">
+  <img src="harbor-logo.png" width="250" title="Harbor Logo">
+</p>
+</br>
+Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. Harbor extends the open source Docker Distribution by adding the functionalities usually required by users such as security, identity and management. Having a registry closer to the build and run environment can improve the image transfer efficiency. Harbor supports replication of images between registries, and also offers advanced security features such as user management, access control and activity auditing.
+</br>
+</br>
+
+- The [contour package](#contour) is a prerequisite for Harbor, so that must be installed
+  first. 
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs-staging.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-EB11DFDE-6ED4-4864-9E60-669F50D64397.html).
+
+
+**Versions:**
+- Download latest version: [Harbor v2.5.3](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=harbor/v2.5.3/harbor.yml) 
+
+**Harbor Sample values.yaml**
+- Download the latest version: [values.yaml sample
+  v2.5.3](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=harbor/v2.5.3/harbor-data-values.yml). For details about each of the required properties, [see the configuration details page](harbor/README.md).
+
+
+## Kubernetes Ingress Controller
+
+</br>
+</br>
+<p align="left">
+  <img src="contour-logo.png" width="250" title="Contour Logo" id="contour">
+</p>
+</br>
+Contour is an Ingress controller for Kubernetes that works by deploying the Envoy proxy as a reverse proxy and load balancer. Contour supports dynamic configuration updates out of the box while maintaining a lightweight profile.
+</br>
+</br>
+
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
+
+
+**Versions:**
+- Download latest version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour.yml) 
+
+**Contour Sample values.yaml**
+- Download the latest version: [values.yaml samplev1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour-data-values.yml). These values can be used _as-is_ and require no configuration changes. 
```

Comment:
Link text looks wrong, I would expect "values for v1.18.2" or the full filename (which ends in .yml not .yaml).